cmake/projects/czmq/hunter.cmake

```diff
@@ -0,0 +1,65 @@
+# !!! DO NOT PLACE HEADER GUARDS HERE !!!
```

Comment:
Please remove this file, only `.travis.yml` or `appveyor.yml` changes:
* https://github.com/ingenue/hunter